docs/apis/file.md

```diff
@@ -0,0 +1,73 @@
+# `file`
+
+The `file` API allows you to retrieve information from any `json` or `csv` file.
+
+* [Basic usage](#Basicusage)
+* [Configuration properties](#Configurationproperties)
+* [Advanced usage](#Advancedusage)
+
+##  <a name='Basicusage'></a>Basic usage
+
+```yaml
+name: example
+apis:
+  - name: linuxDirectorySize
```

Comment:
is this the right value to use for `name` here? this seems like a copy/paste mistake

Reply:
yes, good catch